runtime/onert/backend/cpu/ops/ConvolutionLayer.cc

```diff
@@ -57,10 +57,16 @@ void ConvolutionLayer::convFloat32()
   op_params.float_activation_max = output_activation_max;
 
   nnfw::cker::Conv &kernel = *_conv_kernel;
+
+  const float *kernel_data =
+      (kernel.is_transposed() == true)
+          ? nullptr // _kernel->buffer() is transformed into cker::Conv::_modified_filter_data
```

Comment:
Is it Okay if kernel_data is nullptr?

Reply:
Actually inside the cker::Conv kernel, if it expects transposed weights, it doesn't uses `kernel_data`(whatever comes as parameter).
Instead, it uses _modified_filter_data.

Reply:
And `bool usableMultiThreaded(...)` seems to be working as a 'Do Transpose' flag.

Reply:
Can I get why did you add this variable even though cker::Conv kernel doesn't use `kernel_data`? If the `kernel_data` is not used,  the as-is code seems to be fine.

Reply:
I am working on early-releasing tensor data- releases buffer when tensor's num_reference is zero.
I tried to put assert on buffer() when num_reference is zero.
And this arouse the assert.
I know this is so subtle change... But _transpose, for example, could be fine to use to clarify the meaning here.

Reply:
How about adding an assert for it instead of using `kernel_data`?

Reply:
I talked about this with @dayo09 offline. This line is needed if an assertion is added in `Tensor::buffer()` to check that the tensor's buffer is not nullptr. So we agree to add the code, @dayo09 is going to do it.